### ISSUE TITLE
Promise.done is deprecated.

### ISSUE THREAD

**Issue by trevershick** [jumpkick/graphviz-preview#27]
Atom now uses ES6 Promises instead of Q. Call promise.then instead of promise.done
```
Promise.done (/Applications/Atom.app/Contents/Resources/app.asar/src/atom-environment.js:1139:5)
Object.toggle (/Users/tshick/.atom/packages/graphviz-preview/lib/graphviz-preview.coffee:67:10)
atom-text-editor.subscriptions.add.atom.commands.add.graphviz-preview:toggle (/Users/tshick/.atom/packages/graphviz-preview/lib/graphviz-preview.coffee:17:26)
CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61)
KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
```

**Comment by deftdawg** [comment]
This should be fixed now, commit was a58c852655a65da0107d735b9dd9a8bbe3260123 - graphviz-plugin release 1.7.0

**Comment by whimshot** [comment]
I got the following after installing graphviz-preview

Promise.done - <embedded>:1684:11
Object.toggle - /Users/colinj/.atom/packages/graphviz-preview/lib/graphviz-preview.coffee:67:10
HTMLElement.subscriptions.add.atom.commands.add.graphviz-preview:toggle - /Users/colinj/.atom/packages/graphviz-preview/lib/graphviz-preview.coffee:17:26
CommandRegistry.handleCommandEvent - <embedded>:21847:35
<unknown> - <embedded>:21585:65
KeymapManager.dispatchCommandEvent - <embedded>:66698:22
